query-engine/js-connectors/js/js-connector-utils/src/types.ts

```diff
@@ -3,9 +3,28 @@ import { ColumnTypeEnum } from './const'
 export type ColumnType = typeof ColumnTypeEnum[keyof typeof ColumnTypeEnum]
 
 export interface ResultSet {
+  /**
+   * List of column types appearing in a database query, in the same order as `columnNames`.
+   * They are used within the Query Engine to convert values from JS to Quaint values.
+   */
   columnTypes: Array<ColumnType>
+
+  /**
+   * List of column names appearing in a database query, in the same order as `columnTypes`.
+   */
   columnNames: Array<string>
+
+  /**
+   * List of rows retrieved from a database query.
+   * Each row is a list of values, whose length matches `columnNames` and `columnTypes`.
+   */
   rows: Array<Array<any>>
```

Comment:
```suggestion
  rows: Array<Array<unknown>>
```

I know `any` was there before, so it's fine if you keep it for now.

Reply:
Let's open a new PR for simplifying/improving types